spec/factories/repositories.rb

```diff
@@ -8,5 +8,20 @@
     tracked false
     url "MyString"
     html_url "MyString"
+
+    factory :repository_with_pull_requests_and_reviews do
+      transient do
+        pull_request_count 5
+        reviews_count 3
+      end
+
+      after(:create) do |profile, evaluator|
+        create_list(:pull_request_with_reviews,
+          evaluator.pull_request_count,
+          reviews_count: evaluator.reviews_count,
+          repository: profile
+        )
```

Comment:
Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.